include/deviceconfig.h

```diff
@@ -128,6 +133,10 @@ class DeviceConfig : public IJSONSerializable
         SetIfPresentIn(jsonObject, useCelsius, UseCelsiusTag);
         SetIfPresentIn(jsonObject, youtubeChannelGuid, YouTubeChannelGuidTag);
         SetIfPresentIn(jsonObject, youtubeChannelName1, YouTubeChannelName1Tag);
+        SetIfPresentIn(jsonObject, ntpServer, NTPServerTag);
+
+        if (ntpServer.isEmpty())
+            ntpServer = DEFAULT_NTP_SERVER;
 
         if (jsonObject.containsKey(TimeZoneTag))

```

Comment:
What's the difference between ContainsKey and !isEmpty?  Might be worth a comment.